pandas/tests/internals/test_internals.py

```diff
@@ -1243,7 +1243,6 @@ def test_binop_other(self, op, value, dtype):
                    (operator.mul, '<M8[ns]'),
                    (operator.add, '<M8[ns]'),
                    (operator.pow, '<m8[ns]'),
-                   (operator.mod, '<m8[ns]'),
```

Comment:
?

Reply:
This is testing basically-wrong behavior.